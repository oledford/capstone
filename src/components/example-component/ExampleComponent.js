import React from 'react';
import { Text, View } from 'react-native';
import ExampleComponentStyles from './ExampleComponentStyles';

const ExampleComponent = () => {
  const { container, helloWorld } =  ExampleComponentStyles;

  return (
    <View style={container}>
      <Text>Open up App.js to start working on your app!</Text>
      <Text>Changes you make will automatically reload.</Text>
      <Text>Shake your phone to open the developer menu.</Text>
      <View style={helloWorld}>
        <Text>Hello, World!</Text>
      </View>
    </View>
  );
};

export default ExampleComponent;
