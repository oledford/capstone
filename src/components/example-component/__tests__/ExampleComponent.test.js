import React from 'react';
import ExampleComponent from '../ExampleComponent';

import renderer from 'react-test-renderer';

it ('renders without crashing', () =>{
  const rendered = renderer.create(<ExampleComponent />).toJSON();
  expect(rendered).toMatchSnapshot();
});
