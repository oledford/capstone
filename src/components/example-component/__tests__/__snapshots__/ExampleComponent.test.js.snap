// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "space-between",
      "padding": 100,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Open up App.js to start working on your app!
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Changes you make will automatically reload.
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Shake your phone to open the developer menu.
  </Text>
  <View
    style={
      Object {
        "backgroundColor": "red",
        "margin": 30,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    >
      Hello, World!
    </Text>
  </View>
</View>
`;
